### DISCUSSION TITLE
[13.0] Migration from v12 docker version

### DISCUSSION THREAD

**Discussion by AmeliaPessoa** [OCA/OpenUpgrade#2963] [Q&A]
I opened this issue about the migration errors: [https://github.com/OCA/OpenUpgrade/issues/2962](https://github.com/OCA/OpenUpgrade/issues/2962)

@pedrobaeza you said that I am not running odoo-bin from standard Odoo package. I am not understanding your point because I'm using the `OpenUpgrade-13.0/odoo`. Could you please help me?

**Comment by pedrobaeza** [comment]
I cannot help you more, as it seems `odoo-bin` is not belonging to the OpenUpgrade thing, but the main Odoo one. Use Doodba that is more easy as you only have to point to OpenUpgrade in the repos.yaml, and there I will be more helpful. but let's see if others use the official docker Odoo and can help you.

**Reply by AmeliaPessoa** [reply]
I am using the odoo inside the OpenUpgrade-13.0. Is this not the right way?
